### PR TITLE
List Bitcoin Rhodium (XRC)

### PR DESCRIPTION
**Pull request by ghost**
Official project URL: https://www.bitcoinrh.org/
Official block explorer URL: https://explorer.bitcoinrh.org/

### CODE REVIEW

**Comment by aejontargaryen** [comment]
not even a fork of bitcoin, and uses the name bitcoin. I mean no disrespect to your project, as i know how hard project development can be...  i will say at least its low supply, but damn, this really is **the** _perfect_ example of what ManfredKarrer talks about abusing a name.

File: core/src/main/java/bisq/core/locale/CurrencyUtil.java [hunk 1]
```diff
@@ -145,6 +145,7 @@ public static void setBaseCurrencyCode(String baseCurrencyCode) {
         if (!baseCurrencyCode.equals("LTC"))
             result.add(new CryptoCurrency("LTC", "Litecoin"));
         result.add(new CryptoCurrency("XMR", "Monero"));
+        result.add(new CryptoCurrency("XRC", "Bitcoin Rhodium"));
```
Comment (cbeams) [inline core/src/main/java/bisq/core/locale/CurrencyUtil.java]:
```suggestion
```

File: core/src/test/java/bisq/core/user/PreferencesTest.java [hunk 2]
```diff
@@ -124,7 +124,7 @@ public void testGetUniqueListOfCryptoCurrencies() {
 
         preferences.readPersisted();
 
-        assertEquals(13, preferences.getCryptoCurrenciesAsObservable().size());
+        assertEquals(14, preferences.getCryptoCurrenciesAsObservable().size());
```
Comment (cbeams) [inline core/src/test/java/bisq/core/user/PreferencesTest.java]:
```suggestion
        assertEquals(13, preferences.getCryptoCurrenciesAsObservable().size());
```

**Review by cbeams — CHANGES_REQUESTED** [review]
NACK per comments. Do not add coins / tokens to the list of main cryptocurrencies. Doing so is not part of the instructions at https://bisq.network/list-asset.

**Comment by cbeams** [comment]
> Will be possible to add us to main list when we give you 1000 BTR as donation?

No.

**Pull request merged**